### DISCUSSION TITLE
idea

### DISCUSSION THREAD

**Discussion by unfunnystupididiot** [KadeArchive/Kade-Engine#785] [Ideas]
Not sure if this would be possible to add or not but it'd be cool if you could change what sections have the arrows moving in the debug menu 
(Like how the arrows move at the end of tutorial hard.)

**Comment by isakube** [comment]
arrows don't move in tutorial hard. wdym

**Reply by Lucky-56** [reply]
yes they do, there's a modchart in tutorial

**Reply by isakube** [reply]
i'm a mac user so excuse me for being dumb

**Comment by puyoxyz** [comment]
no! this is the whole point of modchart!! make a modchart!!!

**Reply by ghost** [reply]
can you enable modcharts for Linux and mac please?
im on Linux and it doesnt build to neko it builds to regular hxcpp

**Reply by puyoxyz** [reply]
> can you enable modcharts for Linux and mac please?
> im on Linux and it doesnt build to neko it builds to regular hxcpp

theres an issue for that

**Reply by ghost** [reply]
nvm modcharting breaks the game
next time you update Mic'd Up input system can you disable modcharts on Linux
